__tests__/__snapshots__/react.js.snap

```diff
@@ -5946,6 +6113,48 @@ export default Locals;
 }
 `;
 
+exports[`react-native parses react-native LocalsFormat 1`] = `
+Object {
+  "code": "/* eslint-disable jsx-a11y/accessible-emoji, no-unused-vars */
+import React from 'react';
+import { Text, View } from 'react-native';
+import PropTypes from 'prop-types';
+
+class LocalsFormat extends React.Component {
+  render() {
+    const { context, props } = this;
+
+    return (
+      <View testID={\`\${props['testID'] || 'LocalsFormat'}|\`}>
```

Comment:
The output is missing the local container subscription. Because these views don't have any specific locals, then that's why https://github.com/viewstools/morph/blob/master/morph/react/get-body.js#L3 isn't picking it up. Maybe we need to change that so that we check for this case too and then it will just enable it. Could it be this https://github.com/viewstools/morph/pull/81/files#diff-f139e981b66ed54a2e907c8691b89a27R3?